<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NhapKhoWidget</class>
 <widget class="QWidget" name="NhapKhoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NhapKhoWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="congVanLabel">
       <property name="text">
        <string>Số công văn:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="congVanComboBox"/>
     </item>
     <item>
      <widget class="PhanLoaiTuLieuWidget" name="phanLoaiTuLieuWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="ScanningWidget" name="scanningWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="storeDirLabel">
       <property name="text">
        <string>Nơi lưu trữ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="storeDirLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseStoreDirButton">
       <property name="text">
        <string>Chọn...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progressPathLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="storeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Nhập kho</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PhanLoaiTuLieuWidget</class>
   <extends>QWidget</extends>
   <header>PhanLoaiTuLieuWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScanningWidget</class>
   <extends>QWidget</extends>
   <header>ScanningWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
